Author: Daniel Richard G. <skunk@iSKUNK.ORG>
Description: Don't let GN see {{cc_module_name}}

GN only recently gained support for {{cc_module_name}}:

  ERROR at //build/toolchain/gcc_toolchain.gni:346:17: Unknown substitution pattern
        command = "$cxx -MD -MF $depfile {{defines}} {{include_dirs}} {{cflags}} {{cflags_cc}}${extra_cppflags}${extra_cxxflags} {{module_deps_no_self}} -fmodule-name={{cc_module_name}} -x c++ -Xclang -emit-module -c {{source}} -o {{output}}"
  Found a {{ at offset 491 and did not find a known substitution following it.

Previously, this command used {{label_name}}.

--- a/build/modules/BUILD.gn
+++ b/build/modules/BUILD.gn
@@ -241,7 +241,6 @@ if (use_unified_system_module) {
   }
 
   source_set("system") {
-    module_name = "//build/modules:system"
     sources = [ modulemap ]
     public = system_headers
     use_libcxx_modules = false
--- a/build/toolchain/gcc_toolchain.gni
+++ b/build/toolchain/gcc_toolchain.gni
@@ -322,7 +322,7 @@ template("single_gcc_toolchain") {
     # -fmodule-name is not supported by gcc.
     module_name_flag = ""
     if (is_clang) {
-      module_name_flag = " -fmodule-name=\"{{cc_module_name}}_Private\""
+      module_name_flag = " -fmodule-name=\"{{label_name}}_Private\""
     }
 
     enable_linker_map = defined(invoker.enable_linker_map) &&
@@ -373,7 +373,7 @@ template("single_gcc_toolchain") {
 
       # Module file doesn't need coverage instrumentation because module files
       # represent interfaces rather than implementations.
-      command = "$cxx -MD -MF $depfile {{defines}} {{include_dirs}} {{cflags}} {{cflags_cc}}${extra_cppflags}${extra_cxxflags} {{module_deps_no_self}} -fmodule-name=\"{{cc_module_name}}\" -x c++ -Xclang -emit-module -c {{source}} -o {{output}}"
+      command = "$cxx -MD -MF $depfile {{defines}} {{include_dirs}} {{cflags}} {{cflags_cc}}${extra_cppflags}${extra_cxxflags} {{module_deps_no_self}} -fmodule-name=\"{{label_name}}\" -x c++ -Xclang -emit-module -c {{source}} -o {{output}}"
       depsformat = "gcc"
       description = "CXX_MODULE {{output}}"
       outputs = [ "$object_subdir/{{source_name_part}}.pcm" ]
